rules/windows/process_creation/proc_creation_win_tool_nircmd.yml

```diff
@@ -16,19 +17,25 @@ logsource:
     category: process_creation
     product: windows
 detection:
-    selection:
-        Image|endswith:
-            - '\nircmd.exe'
-            - '\nircmdc.exe'
-    selection_params:
+    selection_org:
+        OriginalFileName|contains: 'NirCmd.exe'
+    selection_execmd:
+        CommandLine|contains: ' execmd '
+    selection_exec:
         CommandLine|contains:
-            - ' execmd '
+            - ' exec '
             - ' exec2 '
-    selection_commands:
+    selection_exec_params:
         CommandLine|contains:
-            - ' copy '
-            - ' del '
-    condition: selection or ( selection_params and selection_commands )
+            - ' show '
+            - ' hide '
+    selection_script:
+        CommandLine|contains: '.exe script '
+    selection_shexec:
+        CommandLine|contains: '.exe shexec '
+    selection_runinteractive:
+        CommandLine|contains: ' runinteractive '
+    condition: 1 of selection*
```

Comment:
As is a `OR` every commandline with a ` hide ` or ` exec ` will trigger.
Do you think there will be not to much FP ?
Thanks

Reply:
So we are looking for 1 of these ?
- OriginalFileName  = NirCmd.exe
- CommandLine ~=  (execmd | exec2) AND (show | hide)
- CommandLine ~=  execmd
- CommandLine ~=  .exe script
- CommandLine ~=  .exe shexec
- CommandLine ~=  runinteractive

all the OR `CommandLine|contains` can merge to :
```yaml
selection_cmd:
  CommandLine|contains: 
    - ' execmd '
    - '.exe script '
    - '.exe shexec '
    - ' runinteractive '
````

Reply:
I've made the necessary changes.